doc/api/tls.md

```diff
@@ -851,12 +851,12 @@ publicly trusted list of CAs as given in
   * `passphrase` {string} A string containing the passphrase for the private
     key or pfx.
   * `cert` {string|string[]|Buffer|Buffer[]} A string, `Buffer`, array of
-    strings, or array of `Buffer`s containing the certificate key of the server
-    in PEM format. (Required)
+    strings, or array of `Buffer`s containing the server's certificate chain
+    (server's certificate and intermediates) in PEM format. (Required)
   * `ca` {string|string[]|Buffer|Buffer[]} A string, `Buffer`, array of strings,
     or array of `Buffer`s of trusted certificates in PEM format. If this is
     omitted several well known "root" CAs (like VeriSign) will be used. These
-    are used to authorize connections.
+    are used to authorize connections. Useful for `requestCert` option.
```

Comment:
That isn't helpful, you need to describe what it does, and why what it does combines well with other options.